app/assets/javascripts/moirai_translation_controller.js

```diff
@@ -6,7 +6,7 @@ export default class MoiraiTranslationController extends Controller {
   }
 
   submit(event) {
-    const {filePath, key} = event.target.dataset
+    const {key} = event.target.dataset
```

Comment:
I believe the key and value should be stimulus values. https://stimulus.hotwired.dev/reference/values

Reply:
The `key` can be a stimulus value, the `value` on the other hand is the dynamic value of the input field. I'll refactor the `key` to be a stimulus value but keep the `value`.